### ISSUE TITLE
Auto Layout to start diagram from centre instead of left

### ISSUE THREAD

**Issue by MohammedAbdulMateen** [dagrejs/dagre#249]
Hi,

Thanks for creating this plugin, I was struggling to create an auto layout for a diagram (questionnaire) which consists of nodes (div's) and edges (SVG's created using jsPlumb) but I tried code from this article http://onais-m.blogspot.com/2014/10/automatic-graph-layout-with-javascript.html

This has done the auto layout for the diagram in a nice way, but the diagram goes from left to right, but I want the diagram to start in the middle and spread left and right as required i.e. like center aligned.

Please help on how to achieve this.

**Comment by ipavlic** [comment]
I don't believe that's supported by the library or by papers the library is using.

**Comment by MohammedAbdulMateen** [comment]
oh is that so... okay

**Comment by atulgirishkumar** [comment]
Hi, @ipavlic again a very nice plugin. We are using it with jointjs. It beautifully spans the graph from left to right, but we are in need of the functionality mentioned by @MohammedAbdulMateen .
We want the graph node to be starting from the center and spanning left and right.

Thanks.

**Comment by mawg** [comment]
Sigh !   And Dagre looked so promising too :-(  

Oh, well, that's the end of that.

Can anyone suggest a library that will layout like that, and also auto-resize after dynamically adding anode to an existing graph?
